apps/ios/SimpleXChat/API.swift

```diff
@@ -239,6 +239,14 @@ private func decodeUser_(_ jDict: NSDictionary) -> UserRef? {
     }
 }
 
+private func errorJson(_ jDict: NSDictionary) -> String? {
```

Comment:
invalidJson errors in ios included full response json including user - this excludes user from invalidJson string

Reply:
wouldn't it exclude the whole thing if it's not error?

Reply:
but it can only be error there

Reply:
theoretically

Reply:
it is also used in chatCmdError

Reply:
probably it's the key higher up... In any case, seems safer to just keep it as is rather than null it, also avoids nullable return

if let chatError = jDict["chatError"] {
         json(chatError)
     } else {
         json(jDict)
     }